src/App.css

```diff
@@ -6,7 +6,7 @@
 
```

Comment:
Might be a good idea to organize this or separate into files. Just a lot of stuff.